From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Thu, 26 Mar 2020 14:52:20 -0500
Subject: [PATCH] Linux 4.14.174-rt79 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c642ed5..cf94a551bf32 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt79
-- 
https://clearlinux.org

